iceoryx_hoofs/test/moduletests/test_cxx_optional.cpp

```diff
@@ -497,82 +493,87 @@ TEST_F(Optional_test, CopyConstructionWithElementWorks)
     EXPECT_THAT(sut->secondValue, Eq(6));
 }
 
+constexpr char DEFAULT_STRING[]{"Live long and prosper"};
+constexpr int8_t DEFAULT_INT{0};
+constexpr int8_t DEFAULT_MULTIPLICATOR{2};
+
 struct TestStructForInPlaceConstruction
 {
-    TestStructForInPlaceConstruction()
-        : TestStructForInPlaceConstruction(0.0, std::make_unique<std::string>("Hello"))
-    {
-    }
+    TestStructForInPlaceConstruction() = default;
```

Comment:
Since this will be used when the copy and move ctors are deleted, wouldn't it make sense to also explicitly delete it for this class?